l10n_th_asset_register_report/report/asset_register.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import models, fields, api
+
+
+class AssetView(models.TransientModel):
+    _name = 'asset.view'
+    _inherit = 'account.asset'
+    _order = 'id'
+
+    asset_id = fields.Many2one(
+        comodel_name='account.asset',
+    )
+    percent_depreciation = fields.Float()
+    depreciation = fields.Float()
+    accumulated_cf = fields.Float()
+    accumulated_bf = fields.Float()
+
+
+class AssetRegisterReport(models.TransientModel):
+    _name = 'report_asset_register'
```

Comment:
Why not name it report.asset.register?
(I just curious, if you have good reason please let me know)

Reply:
it's my mistake